src/Radar.js

```diff
@@ -0,0 +1,237 @@
+import {TextBox} from "d3plus-text";
+
+import {default as Plot} from "./Plot";
+
+const tau = Math.PI * 2;
+
+/**
+    @class Radar
+    @extends Plot
+    @desc Creates a radar visualization based on an array of data.
+*/
+export default class Radar extends Plot {
+
+  /**
+      @memberof LinePlot
+      @desc Invoked when creating a new class instance, and overrides any default parameters inherited from Plot.
+      @private
+  */
+  constructor() {
+    super();
+    this._discrete = "x";
+    this._levels = 6;
+
+    this._on.mouseenter = () => {};
+    this._on["mouseleave.shape"] = () => {
+      this.hover(false);
+    };
+    const defaultMouseMove = this._on["mousemove.shape"];
+    this._on["mousemove.shape"] = (d, i) => {
+      defaultMouseMove(d, i);
+      const id = this._id(d, i);
+
+      this.hover((h, x) => this._x(h, x) === id);
+    };
+
+    this._radarPadding = 100;
+
+    this._shape = constant("Path");
+    this._shapeConfig = assign(this._shapeConfig, {
+      Circle: {
+        r: accessor("r", 0),
+        fill: constant("none"),
+        stroke: constant("#CCC"),
+        strokeWidth: constant(1)
+      },
+      Path: {
+        fillOpacity: 0.7
+      }
+    });
+    this._xConfig = {
+      height: 0
+    };
+    this._value = accessor("value");
+
+    this._y1 = false;
+  }
+
+  /**
+      Extends the draw behavior of the abstract Plot class.
+      @private
+  */
+  _draw(callback) {
+    const height = this._height - this._margin.top - this._margin.bottom,
+          width = this._width - this._margin.left - this._margin.right;
+
+    const radius = (Math.min(height, width) - this._radarPadding) / 2,
+          transform = `translate(${width / 2}, ${height / 2})`;
+
+    const maxValue = Math.max(...this._data.map((d, i) => this._value(d, i))),
+          nestedAxisData = nest()
+        .key(this._y)
+        .entries(this._data),
+          nestedGroupData = nest()
+        .key(this._x)
+        .entries(this._data);
+
+    const circularAxis = Array.from(Array(this._levels).keys()).map(d => ({
+      id: d,
+      r: radius * ((d + 1) / this._levels)
+    }));
+
+    this._shapes.push(
+      new Circle()
+        .data(circularAxis)
+        .select(
+          elem("g.d3plus-Radar-radial-circles", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .config(configPrep.bind(this)(this._shapeConfig, "shape", "Circle"))
+        .render()
+    );
+
+    const totalAxis = nestedAxisData.length;
+    const polarAxis = nestedAxisData
+      .map((d, i) => {
+        const angle = tau / totalAxis * i;
+        return {
+          id: d.key,
+          angle: 360 - 360 / totalAxis * i,
+          quadrant: parseInt(360 - 360 / totalAxis * i / 90, 10) % 4 + 1,
+          x: radius * Math.cos(angle),
+          y: radius * Math.sin(angle)
+        };
+      })
+      .sort((a, b) => a.key - b.key);
+
+    this._shapes.push(
```

Comment:
Only shapes that have user-data attached to them should be added to `this._shape`. Don't push the the axis labels. (this may be related to hover issues)